tests/CacheCompat/CommonCache.Test.MsalJava/pom.xml

```diff
@@ -24,7 +24,7 @@
         <dependency>
             <groupId>com.microsoft.azure</groupId>
             <artifactId>msal4j</artifactId>
-            <version>0.5.0-preview</version>
+            <version>0.6.0-preview</version>
```

Comment:
We should look into version ranges / using LATEST, but as a separate work item, as it does not seem to be trivial.